docs/Clearwater_SNMP_Statistics.md

```diff
@@ -29,6 +29,19 @@ For example, a stat queried at 12:01:33 would display the stats covering:
 
 All latency values are in microseconds.
 
+Many of the statistics listed below are stored in SNMP tables (although the MIB file should be examined to determine exactly which ones). The full table can be retrieved by using the 'snmptable' command. For example, the Initial Registration Stats table for sprout can be retrieved by running:  
+`snmptable -v2c -c clearwater <ip> PROJECT-CLEARWATER-MIB::sproutInitialRegistrationsTable`
+
+The individual table elements can be accessed using:  
+`snmpget -v2c -c clearwater <ip> <table OID>.1.<column>.<row>`
+
+For example, the Initial Registration Stats table has an OID of `.1.2.826.0.1.1578918.9.3.9`, so the number of initial registration attempts in the current five-minute period can be retrieved by:    
+`snmpget -v2c -c clearwater <ip> .1.2.826.0.1.1578918.9.3.9.1.2.3`  
+or by:  
+`snmpget -v2c -c clearwater <ip> PROJECT-CLEARWATER-MIB::sproutInitialRegistrationAttempts.scopePrevious5MinutePeriod`
+
+Note that running an 'snmpget' on a table OID will result in a "No Such Object available on this agent at this OID" message.
```

Comment:
Most of the other errors have more self-explanatory error messages ("Was that a table?" or "Cannot find module") so I'm not sure there's much value in this.